pkg/controller/disruption/disruption_test.go

```diff
@@ -390,30 +415,41 @@ func TestMultipleControllers(t *testing.T) {
 	pdb, pdbName := newPodDisruptionBudget(t, intstr.FromString("1%"))
 	add(t, dc.pdbStore, pdb)
 
+	pods := []*v1.Pod{}
 	for i := 0; i < podCount; i++ {
 		pod, _ := newPod(t, fmt.Sprintf("pod %d", i))
+		pods = append(pods, pod)
 		add(t, dc.podStore, pod)
 	}
+
 	dc.sync(pdbName)
 
 	// No controllers yet => no disruption allowed
 	ps.VerifyDisruptionAllowed(t, pdbName, 0)
 
 	rc, _ := newReplicationController(t, 1)
 	rc.Name = "rc 1"
+	for i := 0; i < podCount; i++ {
+		updatePodOwnerToRc(t, pods[i], rc)
```

Comment:
go test ./pkg/controller/disruption/... --count 300 --race 
race: limit on 8192 simultaneously alive goroutines is exceeded, dying
FAIL	k8s.io/kubernetes/pkg/controller/disruption	16.577s

go test ./pkg/controller/disruption/... --count 250 --race 
ok  	k8s.io/kubernetes/pkg/controller/disruption	16.575s

I cant go beyond 250 on my mac.  Is there a way to request this count with the prbot ?